abs.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+	"io"
+	"log"
+	"os"
+	"strings"
+)
+
+type ABS_Manager struct {
+	*BucketMgr
```

Comment:
Why pointer? Without pointer you can also access the fields of `BucketMgr`

Reply:
Hey thank you.. will update